FormConfig/EmailConfig.php

```diff
@@ -43,4 +46,25 @@ public function getBody(): string
     {
         return $this->body;
     }
+
+    public function getAttachments(): array
+    {
+        return $this->attachments;
+    }
+
+    private function sanitiseBody(string $body): string
+    {
+        $body = preg_replace('/^&quot;|&quot;$/', '', $body);
```

Comment:
No need to create this $body variable, we can immediatly return the preg_replace result.